README.md

```diff
@@ -1,28 +1,56 @@
-== README
+# Hello, fellow developers!
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Welcome to URDb, your go-to source for all things movies related. Whether it be movie ratings, actors, or reviews, you know you can find it all here! (Kind of.)
 
-Things you may want to cover:
+ As successful as we have been so far, we've run into alot of bugs - and because of that, we've hired you to come and write an extensive (ish) test suite to prevent problems in the future. Let's get you set up.
 
-* Ruby version
+## Setup
 
-* System dependencies
+To begin, clone/fork this repository.
 
-* Configuration
+Afterwards, run:
 
-* Database creation
+```
+bundle install
+```
 
-* Database initialization
+to install rspec, our testing framework of choice.
 
-* How to run the test suite
+To create our database, run:
 
-* Services (job queues, cache servers, search engines, etc.)
+```
+rake db:create && rake db:migrate
+```
 
-* Deployment instructions
+Let's make sure our testing framework is working by running
 
-* ...
+```
+rspec
+```
+You should see tests run....and fail. What happened? Let's keep going.
+
+## Part 1
+
+Let's first take a look at what we have so far - it looks like we have a movies model, that has a name, and a rating - see `/app/models/movie.rb`
+
+We've started writing some tests for you - located in `spec/models`. We definitely want to make sure that is data validation before we save our models - unfortunately URDb didn't have the time to actually make the tests pass.
+
+Your job: Add the proper validations to the movie model to make the tests pass.
+
+Hint: Take a look at the rails documentation to see how validations on data are implmented.
+
+## Part 2
+
+Note: Remember that you should be writing your tests before writing that actual implementations of these methods.
```

Comment:
Move this to the bottom? Or separate it from the rest of the text visually in some way? It's a little confusing right here.